core-java/src/test/resources/schema/CreateUserSchema.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
All JSON file names should be lower case with words separated by hyphens.
E.g: this file name will be `create-user-schema.json`

Modify the names for all the JSON schema files.

Reply:
Done